app/src/main/res/values/styles.xml

```diff
@@ -327,4 +327,39 @@
     <item name="android:textAllCaps">false</item>
     <item name="android:textColor">@color/oppiaPrimaryText</item>
   </style>
+  <!-- Start Over Lesson Button -->
+  <style name="StartOverLessonButton" parent="TextAppearance.AppCompat.Widget.Button">
+    <item name="android:layout_width">144dp</item>
+    <item name="android:layout_height">wrap_content</item>
+    <item name="android:paddingRight">12dp</item>
```

Comment:
Please use paddingStart and paddingEnd here and everywhere else in this PR.

Left/Right does not translate well for RTL

Reply:
done